README.md

```diff
@@ -68,6 +86,14 @@ ruby -r vernier -e 'Vernier.trace_retained(out: "irb_profile.json") { require "i
 
 Retained-memory flamegraphs must be interpreted a little differently than a typical profiling flamegraph. In a retained-memory flamegraph, the x-axis represents a proportion of memory in bytes,  _not time or samples_ The topmost boxes on the y-axis represent the retained objects, with their stacktrace below; their width represents the percentage of overall retained memory each object occupies.
 
+### Options
```

Comment:
Wasn't sure if we wanted to add `hooks` here. [The only one I could find](https://github.com/jhawthorn/vernier/blob/main/lib/vernier/hooks/active_support.rb) seems to be more of an private hook for supporting Rails, and [this]https://github.com/jhawthorn/vernier/blob/f8976625d7b5cfeeddb8471750f1ca93cf10d5aa/lib/vernier/collector.rb#L36) implies it's a private API.

Reply:
I imagine having more hooks and user supplied hooks, but I'm not quite sure the best API for it yet

Reply:
Best to hold off on adding it to the README then?